### PR TITLE
write actual prompt for user input

### PR DESCRIPTION
**Pull request by mimmerma**
- decided to print an intro before the input prompt with the details of
user input formats
- kept the prompt itself short and sweet

### CODE REVIEW

File: raw_input.py [hunk 1]
```diff
@@ -1,11 +1,32 @@
 #
```
Comment (jdimmerman) [inline raw_input.py]: Are these three hashtags on the top of the file a python convention?

File: raw_input.py [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
-
 import rlcompleter
 
-# raw_input([prompt]) - no newline after [prompt] in output
-# returns user input
-s = raw_input('--> ')
-print("you entered: ", s)
+# intro
+print
```
Comment (jdimmerman) [inline raw_input.py]: Is there a way to create the string ahead of time (including any linebreaks), store it in a variable, and then print it out?  The many print statements will make the string ultimately harder to manipulate and isn't visually pleasing in my opinion.

Comment (jdimmerman) [inline raw_input.py]: I'd also suggest moving this to a help() or dispaly_help() or print_help() method.  You'll likely want to call it not just here, but when someone enters incorrect info.

File: raw_input.py [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
+print('Note about deathNotice: must be called with name of existing Person object')
+print('Note about deathNotice: date-of-death is optional because might be unknown')
+
+print
+print('displayTree method takes no arguments')
+print('Ex: displayTree')
+print('\n')
+
+# prompt for user input in form described in intro
+userInput = raw_input('[method], [<optional-arguments-comma-delimited>] --> ')
```
Comment (jdimmerman) [inline raw_input.py]: agian, I suggest moving this to its own method parse_input() or something.

File: raw_input.py (hunk 1 above)
Comment (jdimmerman) [inline raw_input.py]: Are these three hashtags on the top of the file a python convention?

Comment (mimmerma) [inline raw_input.py]: No, I've just been adding them at the top and bottom of files to help with readability.  Should I not?

File: raw_input.py (hunk 2 above)
Comment (mimmerma) [inline raw_input.py]: Yes I can store the string in a variable with linebreaks and then print the variable.

Comment (mimmerma) [inline raw_input.py]: Good point about the help() method and calling it repeatedly.

File: raw_input.py (hunk 3 above)
Comment (mimmerma) [inline raw_input.py]: I had moved both of the codeblocks (the intro and input_prompt) into functions in the next change.

File: raw_input.py (hunk 1 above)
Comment (jdimmerman) [inline raw_input.py]: I dunno, its up to you.  You'll be asked about it so I'd ultimately suggest not but its not a big deal either way.

File: raw_input.py (hunk 3 above)
Comment (jdimmerman) [inline raw_input.py]: Weird that you can add emojis to github comments...

Comment (mimmerma) [inline raw_input.py]: It's an emotional process

**Pull request merged**